extensions-core/parquet-extensions/src/main/java/org/apache/druid/data/input/parquet/simple/ParquetGroupConverter.java

```diff
@@ -364,6 +364,7 @@ private static Object convertPrimitiveField(Group g, int fieldIndex, int index,
           case UINT_16:
             return g.getInteger(fieldIndex, index);
           case UINT_32:
+            return g.getInteger(fieldIndex, index) & (-1L >>> 32);
```

Comment:
this could also use `Integer.toUnsignedLong`

Reply:
Fixed it, Thanks.